code/datums/components/storage/storage.dm

```diff
@@ -0,0 +1,17 @@
+
+/datum/component/storage
+	dupe_mode = COMPONENT_DUPE_UNIQUE
+
+/datum/component/storage/concrete
```

Comment:
why this subtype?

Reply:
Only to mirror what tg is doing, no other reason.
Should I remove it if it's not serving any real purpose?